components/mediation/registry/org.wso2.micro.integrator.registry/src/main/java/org/wso2/micro/integrator/registry/MicroIntegratorRegistry.java

```diff
@@ -1498,7 +1508,7 @@ private void addImmediateChildren(File node, JSONArray childArray, String carbon
      * @param nodeName  Node name
      * @return          Boolean output of the checks
      */
-    private boolean isNodeNotRequiredToBeFetched(String nodeName) {
+    public boolean isNodeNotRequiredToBeFetched(String nodeName) {
```

Comment:
Any particular reasons to make this function public?

Reply:
Resolved, yes it should be private.